src/components/NewsCarouselItem/NewsCarouselItem.tsx

```diff
@@ -0,0 +1,33 @@
+import { Grid } from '@trussworks/react-uswds'
+import React from 'react'
+import styles from './NewsCarouselItem.module.scss'
+import { ArticleListItemRecord } from 'types'
+import LinkTo from 'components/util/LinkTo/LinkTo'
+
+const NewsCarouselItem = ({ article }: { article: ArticleListItemRecord }) => {
+  return (
+    <Grid row col="auto">
+      <Grid col={5}>
+        <img
```

Comment:
TODO: Add comment here with a, uh, TODO, reminding future us that this image is a placeholder and will get replaced with cms data.